src/OpenXR/Silk.NET.OpenXR/NativeWindowExtensions.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+using Silk.NET.Core;
+using Silk.NET.Core.Contexts;
+
+namespace Silk.NET.OpenXR
+{
+    public static class NativeWindowExtensions
+    {
+        public static GraphicsBinding CreateOpenGLBinding<T>(this T src)
+            where T : INativeWindowSource, IGLContextSource
```

Comment:
Should I know what INativeWindowSource is? lmao

Reply:
Is this a review comment?

Reply:
Quick explainer what this does/where it's used? I see this type being used the first time in our codebase? seems like it's OpenXR specific?

Reply:
INativeWindowSource is an existing type which is used to describe a type that can be represented using a INativeWindow, an interface for getting platform-specific window handles from an object representing a window.